test/fees/FeeGeneralCases.t.sol

```diff
@@ -115,6 +118,46 @@ contract FeeRevertCasesTest is Test {
         router.execute{value: nativeAmount}(logics, tokensReturnEmpty, SIGNER_REFERRAL);
     }
 
+    function testSetFeeCalculator(bytes4 selector, address to, address feeCalculator) external {
```

Comment:
Should test `onlyOwner` so need to add `testSetFeeCalculatorByNonOwner` and `testSetFeeCalculatorsByNonOwner`

Reply:
Added. Saw other `xxxByNonOwner` tests  use `Cannot` prefix, so add `Cannot` as well.